### PR TITLE
Add Global Azure CLI config flag

### PR DESCRIPTION
**Pull request by vineetmimrot**
This commit will add Use global Azure CLI configuration flag. This will enable the user to run AzCLI task in parallel releases without conflicting with each other.

### CODE REVIEW

File: Tasks/AzureCLIV1/azureclitask.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export class azureclitask {
                 tool = tl.tool(tl.which(scriptPath, true));
             }
             this.throwIfError(tl.execSync("az", "--version"));
+            this.useGlobalConfig = tl.getBoolInput("useGlobalConfig");
```
Comment (bishal-pdMSFT) [inline Tasks/AzureCLIV1/azureclitask.ts]: tl.getBoolInput("useGlobalConfig") || true

Comment (bishal-pdMSFT) [reply Tasks/AzureCLIV1/azureclitask.ts]: We want it to be true for existing RDs

Comment (vineetmimrot) [reply Tasks/AzureCLIV1/azureclitask.ts]: done.

Comment (sachinma) [reply Tasks/AzureCLIV1/azureclitask.ts]: @bishal-pdMSFT  the syntax you have will always return true. I think this probaby takes a default value

Comment (bishal-pdMSFT) [reply Tasks/AzureCLIV1/azureclitask.ts]: yes, you right. the expression is wrong. But we want to be false when it is not set

Comment (vineetmimrot) [reply Tasks/AzureCLIV1/azureclitask.ts]: tl.getBoolInput() will return whatever default value present in task.json.

**Pull request merged**